turingschool/savile
# Add more Helvetica Neue fonts from assets.turing.io

#### What does this PR do?

Allow for a much larger range of font variants in Helvetica Neue

- Use assets.turing.io instead of local font files
- Use progressive font formats (woff2, woff, and true type)
- Enable all Helvetica Neue weights (200-900), in boht italic and normal


#### Where should the reviewer start?
New SCSS file.


#### How should this be manually tested?
Try using a different weight/style font and make sure it loads from the correct font file!


#### Any background context you want to provide?
Relies on font files existing in S3 bucket: [Add our font files to AWS S3 asset bucket](https://3.basecamp.com/3494409/buckets/19192671/todos/3314933961)


#### What are the relevant tickets?

:basecamp: [Replace the fonts from the savile site to use the assets from above](https://3.basecamp.com/3494409/buckets/19192671/todos/3314935459/)


#### Any other deploy steps?
no
